### PR TITLE
Added documentation to PSR1.Methods.CamelCapsMethodName sniff

### PR DESCRIPTION
**Pull request by elplaza**
Created PSR1.Methods.CamelCapsMethodName docs for completeness.

### CODE REVIEW

File: src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<documentation title="Method Name">
+    <standard>
+    <![CDATA[
+    Method names MUST be declared in camelCase().
```
Comment (jrfnl) [inline src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml]:
```suggestion
    Method names MUST be declared in camelCase.
```

File: src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: camel case used.">
```
Comment (jrfnl) [inline src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml]:
```suggestion
        <code title="Valid: method name in camelCase.">
```

File: src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <![CDATA[
+class Foo
+{
+    private function <em>doBar</em>()
+    {
+    }
+}
+        ]]>
+        </code>
+        <code title="Invalid: snake case used.">
```
Comment (jrfnl) [inline src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml]:
```suggestion
        <code title="Invalid: method name not in camelCase.">
```

Comment (elplaza) [reply src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml]: Done!

File: src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml (hunk 2 above)
Comment (elplaza) [reply src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml]: Done!

File: src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml (hunk 1 above)
Comment (elplaza) [reply src/Standards/PSR1/Docs/Methods/CamelCapsMethodNameStandard.xml]: Done!

**Comment by gsherwood** [comment]
Thanks a lot for submitting this

**Pull request merged**